src/components/app/albumsongscontainer/songinfo/song-info.jsx

```diff
@@ -0,0 +1,60 @@
+import React, { Component } from 'react'
+import { observer, inject } from 'mobx-react'
+import { VictoryChart, VictoryBar, VictoryTheme, VictoryPolarAxis } from 'victory'
+
+@inject('songsStore')
+@observer
+class SongInfo extends Component {
+  render() {
+    const { songsStore } = this.props
+
+    const sampleData = [
+      { feature: 'acousticness', value: songsStore.songToDisplay.acousticness },
+      { feature: 'danceability', value: songsStore.songToDisplay.danceability },
+      { feature: 'energy', value: songsStore.songToDisplay.energy },
+      { feature: 'instrumentalness', value: songsStore.songToDisplay.instrumentalness },
+      { feature: 'liveness', value: songsStore.songToDisplay.liveness },
+      { feature: 'speechiness', value: songsStore.songToDisplay.speechiness },
+      { feature: 'valence', value: songsStore.songToDisplay.valence }
+    ]
+
+    return (
+      <div className="si__container">
+        <VictoryChart
+          polar
+          theme={VictoryTheme.material}
+          padding={30}
+          width={220}
+          height={220}
+        >
+          {['acousticness', 'danceability', 'energy', 'instrumentalness',
```

Comment:
Możesz wykorzystać sampleData/labels zamiast tworzyć nowy array i wpisywać to samo w dwóch miejscach

Reply:
Na poczatku tak mialem XD. Potem mialem mega dziwny problem z tym komponentem. Lokalnie bylo ok - a na heroku renderowal sie 7 razy ten sam element i probowalem rozne rzeczy. Okazalo sie finalnie ze paczki cos sie pogryzly i dodanie package-lock pomoglo dopiero. Nawet otwrzylem im issue z tym na githubie hahah. Poprawilem

Reply:
Cool ;)